src/main/scala/org/ergoplatform/nodeView/history/extra/ExtraIndexer.scala

```diff
@@ -208,11 +207,10 @@ trait ExtraIndexerBase extends ScorexLogging {
         findAndUpdateTree(hashErgoTree(box.ergoTree), Right(boxes(boxId)))
 
         // check if box is creating a new token, if yes record it
-        if(tokenRegistersSet(box))
-          cfor(0)(_ < box.additionalTokens.length, _ + 1) { j =>
-            if (!tokens.exists(x => java.util.Arrays.equals(x._1, box.additionalTokens(j)._1)))
-              general += IndexedToken.fromBox(box, j)
-          }
+        cfor(0)(_ < box.additionalTokens.length, _ + 1) { j =>
```

Comment:
I completely removed tokenRegistersSet because tokens can be created without setting registers or something other than token information can be in them

Reply:
That is what I mean, "tokens can be created without setting registers or something other than token information can be in them" should be in ScalaDoc for the method. Know Your Assumptions.

Reply:
Added in ScalaDoc